cmd/talosctl/cmd/mgmt/gen/secureboot.go

```diff
@@ -43,22 +43,18 @@ var genSecurebootUKICmd = &cobra.Command{
 	Long:  ``,
 	Args:  cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return generateSigningCerts(genSecurebootCmdFlags.outputDirectory, "uki", genSecurebootUKICmdFlags.commonName, 4096)
+		return generateSigningCerts(genSecurebootCmdFlags.outputDirectory, "uki", genSecurebootUKICmdFlags.commonName, 4096, true)
 	},
 }
 
-var genSecurebootPCRCmdFlags struct {
-	commonName string
-}
-
 // genSecurebootPCRCmd represents the `gen secureboot pcr` command.
 var genSecurebootPCRCmd = &cobra.Command{
 	Use:   "pcr",
-	Short: "Generates a certificate which is used to sign TPM PCR values",
+	Short: "Generates a key which is used to sign TPM PCR values",
 	Long:  ``,
 	Args:  cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return generateSigningCerts(genSecurebootCmdFlags.outputDirectory, "pcr", genSecurebootPCRCmdFlags.commonName, 2048)
+		return generateSigningCerts(genSecurebootCmdFlags.outputDirectory, "pcr", "dummy", 2048, false)
```

Comment:
`dummy` intentional?

Reply:
yes, because we actually only need the key, there is no cert, but our library always goes through cert & key generation